internal/controller/category/category.go

```diff
@@ -19,7 +19,11 @@ package category
 import (
 	"context"
 	"fmt"
+	"strconv"
 
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/hermsi1337/go-magento2/pkg/magento2/api"
+	"github.com/hermsi1337/go-magento2/pkg/magento2/categories"
```

Comment:
@ca7alindev No,no. This not good, don't use not verified packages. This is no pet project just for fun. Please use pure API calls.